simple/cq_data.c

```diff
@@ -481,7 +481,7 @@ int main(int argc, char **argv)
 
 	hints->ep_attr->type = FI_EP_MSG;
 	hints->caps = FI_MSG;
-	hints->mode = FI_LOCAL_MR | FI_PROV_MR_ATTR;
+	hints->mode = FI_LOCAL_MR | FI_PROV_MR_ATTR | FI_RX_CQ_DATA;
```

Comment:
That said, since this test is using `fi_senddata` as opposed to something like `fi_writedata`, I think the mode bit is irrelevant here since a receive must be posted in any case.

The mode bit should however be added to the `fi_msg_rma` test if it hasn't already.